### PR TITLE
SVG icons for various fediverse systems

### PR DESCRIPTION
**Pull request by annando**
This PR adds a list of SVG icons for nearly all known fediverse systems. This replaces the "Fork Awesome" symbols for these systems.

This PR is currently a draft since adding licensew information for all these files is quite a heavy task.

### CODE REVIEW

**Comment by Art4** [comment]
@annando Can't you using the REUSE tool for licensing? https://reuse.software/dev/

**Comment by annando** [comment]
@Art4 sure, `reuse` tells me, where I have to add a license. But then I have to investigate which licenses apply.

**Comment by tobiasd** [comment]
> @annando Can't you using the REUSE tool for licensing? https://reuse.software/dev/

We use it for our license check, but it does only check for a license for the file in the repository, not for *finding the source of the file and check there for the information* (which would be really cool feature I should really request x)). Sadly this information on other projects repos is well hidden.

**Comment by Art4** [comment]
OK, I understand the problem. Thank you for the explanation.

**Comment by annando** [comment]
I now switched to mostly icons from Wikimedia. There is always license information. "Only" thing I have to do now is to understand how this has to assigned.

**Review by MrPetovan — APPROVED** [review]
This looks good to me, although I'm not a copyright lawyer.

File: images/platforms/diaspora.svg.license [hunk 1]
```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: Matthias Pfefferle https://github.com/pfefferle/openwebicons?ref=svgrepo.com
+SPDX-License-Identifier: CC0-1.0
```
Comment (Art4) [inline images/platforms/diaspora.svg.license]: Missing line break
```suggestion
SPDX-License-Identifier: CC0-1.0

```

File: images/platforms/hubzilla.svg.license [hunk 2]
```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: Matthias Pfefferle https://github.com/pfefferle/openwebicons?ref=svgrepo.com
+SPDX-License-Identifier: CC0-1.0
```
Comment (Art4) [inline images/platforms/hubzilla.svg.license]: Missing line break
```suggestion
SPDX-License-Identifier: CC0-1.0

```

**Review by Art4 — CHANGES_REQUESTED** [review]
Great work. :+1: Just two line breaks are missing.

File: images/platforms/hubzilla.svg.license (hunk 2 above)
Comment (annando) [reply images/platforms/hubzilla.svg.license]: Changes

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**